src/main.rs

```diff
@@ -7,16 +7,16 @@ fn main() {
 
     let args = env::args().skip(1).collect::<Vec<String>>();
 
-    let cartridge_name = match args.first() {
-        Some(name) => {
-            println!("loading {name}");
-            name.clone()
-        }
-        None => {
+    let cartridge_name = args.first().map_or_else(
+        || {
```

Comment:
> Why this change? No problem about it, just for me it is more readable the first version. I don't know if there is some kind improvement

Same.

Reply:
More idiomatic, and also clippy complains :D